src/main/java/org/prgrms/kdt/voucher/Voucher.java

```diff
@@ -0,0 +1,46 @@
+package org.prgrms.kdt.voucher;
+
+import org.prgrms.kdt.voucher.utils.VoucherValidator;
+
+import java.util.Objects;
+
+public class Voucher {
+
+    private VoucherType voucherType;
+    private VoucherAmount amount;
+
+    public Voucher(VoucherType voucherType, VoucherAmount amount) {
+        this.voucherType = voucherType;
+        this.amount = amount;
+    }
+
+    public void validate(VoucherValidator voucherValidator) {
+        voucherValidator.validateAmount(voucherType, amount);
+    }
+
+    public VoucherType getVoucherType() {
+        return voucherType;
+    }
+
+    public VoucherAmount getAmount() {
+        return amount;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof Voucher)) return false;
+        Voucher voucher = (Voucher) o;
+        return voucherType == voucher.voucherType && amount.equals(voucher.amount);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(voucherType, amount);
+    }
+
+    @Override
+    public String toString() {
+        return "[Type]: " + voucherType.getType() + ", [amount]: " + amount.toString();
+    }
```

Comment:
`toString`은 호출하지 않아도 될 것 같아요

```suggestion
    @Override
    public String toString() {
        return "[Type]: " + voucherType.getType() + ", [amount]: " + amount;
    }
```

Reply:
찾아보니 문자열과 객체를 더하거나 객체 자체를 출력하면 `toString` 이 자동으로 호출되네요!